test/multivariate/multinomial.jl

```diff
@@ -1,6 +1,6 @@
 # Tests for Multinomial
 
-using Distributions, Random, StaticArrays
+using Distributions, Random, StaticArrays, ForwardDiff
```

Comment:
Not needed for the changes below it seems:
```suggestion
using Distributions, Random, StaticArrays
```

Reply:
I'll remove it happily, as I don't personally think it really belongs here.
But at least for completeness sake, here is the source of this addition:
https://github.com/JuliaStats/Distributions.jl/pull/1033#issuecomment-570786766

Reply:
It seems it was added because someone wanted dual numbers to be tested as well - but later it was figured out that for other reasons even with the changes in the PR they don't work and the tests were removed again.

So it seems fine to remove this import.